### PR TITLE
Refs #23528 -- Made cosmetic edits to swappable_dependency() docs.

### PR DESCRIPTION
**Pull request by felixxm**

### CODE REVIEW

**Review by smithdc1 — APPROVED** [review]
Super. 👍

Sorry I didn't get chance to finish my review before the previous PR got merged.

File: docs/topics/migrations.txt [hunk 1]
```diff
@@ -224,14 +224,14 @@ Swappable dependencies
 .. function:: django.db.migrations.swappable_dependency(value)
 
 The ``swappable_dependency()`` function is used in migrations to declare
-"swappable" dependencies on migrations in the app of the swapped-in model.
-Currently, the first migration of this app and as a consequence the swapped-in
+"swappable" dependencies on migrations in the app of the swapped-in model,
+currently, the first migration of this app. As a consequence, the swapped-in
```
Comment (felixxm) [inline docs/topics/migrations.txt:228]: @smithdc1 I added "on", is it still good? :thinking:

Comment (smithdc1) [reply docs/topics/migrations.txt]: I think so.

I'd read "part of" or "on" as the same meaning here. 

Does that help?

**Review by shaib — APPROVED** [review]
Yep, this is clearer.

**Comment by felixxm** [comment]
Thanks both :+1:

**Pull request merged**